css/main.scss

```diff
@@ -4,13 +4,13 @@
 @import "variables";
 
 body {
-  background-color: lighten($cream, 2%);
-  font-family: 'Lora', serif;
-  color: $navy;
+  background-color: $gray;
+  font-family: 'Nunito', sans-serif;
+  color: $navy19;
 }
 
 h1 {
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Dosis', sans-serif;
```

Comment:
We only use Dosis for our lockup (i.e. "HackBeanpot" with logo). So all headings should be Nunito!

Reply:
thanks, fixed now!